content/posts/0005-dotnet-rearchitecture.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Introducing the Paketo .NET Core Buildpack v0.1.0
+date: "2021-02-09"
+slug: dotnet-core-rearchitecture
+author: fgallinajones
+---
+
+SOME PICTURE HERE
+
+Paketo is pleased to unveil our new and improved .NET Core Buildpack! With the
+release of [Paketo .NET Core Buildpack
+0.1.0](https://github.com/paketo-buildpacks/dotnet-core/releases), you’ll enjoy
+a host of new features and improvements that we hope will make containerizing
+your .NET apps a breeze.
+
+With the new buildpack, you can now:
+*   Build .NET 5 apps
+*   Provide your app as source code, a framework-dependent deployment (FDD), a
+    framework-dependent executable (FDE), or a self-contained deployment
+*   Build Visual Basic apps
+*   Have source code removed from final app image. Your app container will
+    contain only the built artifacts
+*   Use a
+    [Procfile](https://paketo.io/docs/buildpacks/configuration/#procfiles) to
+    specify custom start processes
+
+We also re-wrote the buildpacks using
+[packit](https://github.com/paketo-buildpacks/packit), our library for
+buildpack developers. This will ensure that the buildpack benefits from future
+innovations in the [Cloud Native Buildpacks
+specification](https://github.com/buildpacks/spec/blob/main/buildpack.md).
+
+If you’re not sure why you should get excited about .NET 5, check out this
+[post from
+Microsoft](https://devblogs.microsoft.com/dotnet/announcing-net-5-0/)
+announcing its release.
+
+&lt; maybe include some highlights about .NET 5?>
+
+It’s easy to get started with the .NET Core Buildpack. In a matter of minutes,
```

Comment:
Maybe add a heading to this section ("Demo") and re-word the intro a little? Suggestion:
"Let's do a quick demo using a .NET 5 app with a React frontend to see how easy it is to get started with the .NET Core Buildpack. In a matter..."